PetStoreTests/LINQUnitTests.cs

```diff
@@ -9,7 +9,7 @@ namespace PetStoreTests
     public class LINQUnitTests
     {
         /// <summary>
-        /// Given a list of 3 pet names, When that list is forward alphabetically (A - Z) sorted Then the returned output should be a reversed sorted list (Z - A)
+        /// Given a list of 3 pet names, When that list is forward alphabetically (A - Z) sorted, Then the returned output should be a reversed sorted list (Z - A)
```

Comment:
I would consider putting the GIVEN WHEN THEN on separate lines to call it out.  eg:

GIVEN a list of 3 pet names
WHEN that list is forward alphabetically sorted
THEN the returned output should be....